src/sonic-ctrmgrd/ctrmgr/ctrmgrd.py

```diff
@@ -581,7 +588,7 @@ def on_state_update(self, key, op, data):
 
     def do_tag_latest(self, feat, docker_id, image_ver):
         ret = kube_commands.tag_latest(feat, docker_id, image_ver)
-        if ret != 0:
+        if ret == 1:
```

Comment:
I suppose 0 means success, other values means failure, right ? what's other values besides 0 and 1 ? why not retry in that  case ?

Reply:
Here has another code -1, it means the scheduled container is not stable, for this case, we don't do tag latest and clean up images